src/transform/LazyLoadTransform.js

```diff
@@ -190,8 +191,7 @@ const loadImage = (document, image) => {
  * @return {!HTMLImageElement[]} Convertible images descendent from but not including element.
  */
 const queryLazyLoadableImages = element =>
-  Array.prototype.slice.call(element.querySelectorAll('img'))
-    .filter(image => isLazyLoadable(image))
+  Polyfill.querySelectorAll(element, 'img').filter(image => isLazyLoadable(image))
```

Comment:
Thoughts on maybe extending prototype on HTMLElement with something like `wmf_querySelectorAll` so don't have to pass `element`? Was wondering about this earlier...